src/main/java/nextstep/ladder/domain/Lines.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.ladder.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lines {
+
+    private final List<Line> lines;
+
+    private Lines(List<Line> lines) {
+        validate(lines);
+        this.lines = lines;
+    }
+
+    private void validate(List<Line> lines) {
+        if (lines == null || lines.isEmpty()) {
+            throw new IllegalArgumentException("사다리 높이는 1 이상이어야 합니다.");
+        }
+    }
+
+    public static Lines of(int numberOfPlayers, int height) {
+        List<Line> lines = IntStream.range(0, height)
+                .mapToObj(i -> LineBuilder.buildWithRandomPoints(numberOfPlayers))
+                .collect(Collectors.toList());
+        return new Lines(lines);
```

Comment:
제가 자바를 안쓴지 오래되서 가물 가물하지만.... 아래와 같이 한번에 사용할 수 있어요.
```suggestion
 .collect(collectingAndThen(toList(), Lines::new)
```